ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleManager.java

```diff
@@ -24,7 +24,7 @@
 
   private static VungleManager sInstance;
 
-  private ConcurrentHashMap<String, VungleBannerAd> mVungleBanners;
+  private ConcurrentHashMap<String, VungleBannerAdapter> mVungleBanners;
```

Comment:
I know that VungleBannerAdapter is not a real adapter, but Google didn't want to have any strong reference to adapter from this map.
We need to be sure that VungleBannerAdapter don't hold any strong reference to real VungleInterstitialAdapter